earthpy/tests/test_spatial.py

```diff
@@ -1,5 +1,88 @@
 from earthpy import spatial
+from osgeo import gdal
+from osgeo import osr
+import rasterio as rio
+import numpy as np
 
+# a helper function to generate a 3D array of data
+def createArray(arr_shape):
+    """ createArray(arr_shape) creates a numpy ndarray with the shape defined by the tuple arr_shape
+        arr_shape should be:
+        (rows, cols, [channels])
+        the values for each channel will be the 1-based number of that channel.
+    """
+    
+    dummy_arr = np.ones(arr_shape)
+    
+    if len(arr_shape) == 2:
+        
+        # there is only one channel, so that one channel is all 1's
+        dummy_arr = np.expand_dims(dummy_arr,-1)
+    
+    else:
+        # multiply each channel by the channel number (1-based)
+        for i in range(arr_shape[-1]):
+            dummy_arr[:,:,i] *= i+1
+    
+    return dummy_arr
+
+# a helper function to write a 3D array of data to disk as a GeoTIFF 
+# the result will have no spatial info or relevance
+def dummyTifWriter(arr, dst_filename):
```

Comment:
Let's also follow docstring conventions. 

```
    """Writes a tif file to a specified location using an array.

    Parameters
    ----------
    arr : array 
        More here on the array.
    dst_filename: path to file...
        More here on the path
```
Please see rasterio docs for examples!
Also can we please use destfile as the argument? i've been using it elsewhere.